### PR TITLE
docs: document new setPublicDataForUser API

### PR DESCRIPTION
**Pull request by lksnmnn**
https://github.com/blitz-js/blitz/pull/2473Cheers,

this PR adds documentation for the `setPublicDataForUser` (naming pending) API.
The API can be used to update session data for other user's to reflect permission/role changes directly, without the need to sign out and in again.

Feature PR: https://github.com/blitz-js/blitz/pull/2473

### CODE REVIEW

**Comment by lksnmnn** [comment]
Vercel only shows me a 404. Anyone can see what goes wrong?

**Comment by flybayer** [comment]
@lksnmnn 
image

File: app/pages/docs/auth-utils.mdx [hunk 1]
```diff
@@ -208,3 +208,28 @@ export const authenticateUser = async (
   return rest
 }
 ```
+
+## `setPublicDataForUser()` {#setPublicDataForUser}
```
Comment (flybayer) [inline app/pages/docs/auth-utils.mdx:212]:
```suggestion
## `setPublicDataForUser()` {#set-public-data-for-user}
```

**Comment by flybayer** [comment]
thank you!!

**Pull request merged**